superset/assets/src/visualizations/core/components/SuperChart.jsx

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
```

Comment:
It might be worth considering using [`react-loadable](https://github.com/jamiebuilds/react-loadable)` for this component. I've used this in some other projects and it provides basically the exact loading functionality we have here.

Not a blocker but thought I'd throw it out there.

Reply:
Your wish is granted.